neuromation/api/jobs.py

```diff
@@ -341,7 +341,15 @@ def __init__(self, core: _Core, config: _Config) -> None:
                 "params": {"job": id, "port": job_port},
             }
         )
-        proxy_command = ["ssh"]
+        proxy_command = [
```

Comment:
if these changes didn't help much (https://github.com/neuromation/platform-client-python/pull/1135/files#r345802769) , do we need them?

Reply:
They help a little maybe?
A little is better than nothing.